data/projects.yml

```diff
@@ -316,3 +316,16 @@ project_list:
     github_user:
       - tipeslowly
     note: "*Clouds of Ink, Pools of Colour* was developed as a snapshot of the print publication of Hou Beiren's work, and as a catalogue for a series of his paintings recently donated to UMAG."
+
+  - id: "shifting_perspectives"
+    src: /screenshots/_shifting_perspectives.jpg
+    alt: "Shifting Perspectives: Stephanie Hanor, Simone Gage, Sage Gaspar, Sydney Pearce, Melika Sebihi, and Taya Wyatt"
+    type: Exhibition Catalogue
+    publisher: Mills College Art Museum
+    title_info: "*Shifting Perspectives*"
+    pub_date: 2020-12-02
+    url: https://mcam.mills.edu//publications/shiftingperspectives/index.html
```

Comment:
There seems to be an extra slash in the URL between edu and publications?

Reply:
Oops! Thank you! I will fix that now.